tests/unit/dojson/test_dojson_hep.py

```diff
@@ -1030,9 +1030,9 @@ def test_preprint_date(marcxml_to_json, json_to_marc):
             json_to_marc['269'][0]['c'])
 
 
-def test_page_nr(marcxml_to_json, json_to_marc):
-    """Test if page_nr is created correctly."""
-    assert (marcxml_to_json['page_nr'][0] ==
+def test_number_of_pages(marcxml_to_json, json_to_marc):
+    """Test if number_of_pages is created correctly."""
+    assert (marcxml_to_json['number_of_pages'][0] ==
```

Comment:
As mentioned IRL please rewrite this test to follow the more extended and useful pattern down below in the code base.